Sources/PackageModel/UserToolchain.swift

```diff
@@ -56,7 +56,7 @@ public final class UserToolchain: Toolchain {
   /// Search paths from the PATH environment variable.
   let envSearchPaths: [AbsolutePath]
 
-  /// Only use search paths, do not fall back to `xcrun`.
+  /// Only use search paths, do not fall back to `xcrun` or `vswhere`.
   let useXcrun: Bool
```

Comment:
It's still undecided if we want to rename the variable. I didn't do this at the point because neither have I got a short and clear name (`useExternalTool` sounds verbose and confusing), nor am I going to make `vswhere` the default like `xcrun` is now. It can be prioritized in later pitches.